src/admin/components/SearchResults.js

```diff
@@ -1,26 +1,14 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
 import { sprintf } from 'jed';
 
 import { gettext as _ } from 'core/utils';
 import SearchResult from 'admin/components/SearchResult';
+import SearchResults from 'amo/components/Search/SearchResults';
 
-import 'admin/css/SearchResults.scss';
+import 'amo/css/SearchResults.scss';
 
 
-export default class SearchResults extends React.Component {
-  static propTypes = {
-    count: PropTypes.number,
-    loading: PropTypes.bool,
-    query: PropTypes.string,
-    results: PropTypes.arrayOf(PropTypes.object),
-  }
-
-  static defaultProps = {
-    count: 0,
-    query: null,
-    results: [],
-  }
-
+export default class AdminSearchResults extends SearchResults {
```

Comment:
Is there a way to support this without inheritance? Could the admin `<SearchPage />` pass in it's special stuff to the common component?

If we can use composition instead of inheritance I would prefer it (sorry I should have commented earlier 😞)